Package.swift

```diff
@@ -15,7 +15,7 @@ let package = Package(
                .upToNextMajor(from: "8.0.0")),
```

Comment:
```suggestion
               .upToNextMajor(from: "8.22.0")),
```

I've been setting all the min versions to the earliest version with full privacy manifest support